core-web/apps/dotcms-ui/src/app/view/components/_common/dot-textarea-content/dot-textarea-content.component.ts

```diff
@@ -45,9 +45,18 @@ export class DotTextareaContentComponent implements OnInit, ControlValueAccessor
     @Input()
     width: string;
 
+    @Input()
+    hideBorder: boolean;
+
+    @Input()
+    editorName: string;
```

Comment:
What happen if is not passed?

Reply:
if it is not passed. It is showing undefined in Monaco Editor Initialization Object. `this.monacoInit.emit({ name: this.editorName, editor });`